web_readonly_bypass/__openerp__.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Authors: Nemry Jonathan & Laetitia Gangloff
+#    Copyright (c) 2014 Acsone SA/NV (http://www.acsone.eu)
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': 'Read Only ByPass',
+    'version': '1.0',
+    "author": "ACSONE SA/NV, Anybox",
+    "maintainer": "ACSONE SA/NV, Odoo Community Association (OCA)",
+    "website": "http://www.acsone.eu",
+    'category': 'Technical Settings',
+    'depends': [
+        'base',
+        'web',
+    ],
+    'description': """
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
You can still put this into README.rst (although duplicated) for easy reading when browsing in GitHub